npm/design-system/src/components/virtualizedTree/VirtualizedTree.tsx

```diff
@@ -30,20 +30,20 @@ const VirtualizedTreeContents = <
   TLeaf extends LeafTreeBase,
   TParent extends ParentTreeBase<TLeaf>
 >({
-    innerRef,
-    treeRef,
-    tree,
-    defaultItemSize,
-    overscanCount = 20,
-    indentSize,
-    showRoot,
-    shouldMeasure,
-    onNodePress: externalOnNodePress,
-    onNodeKeyDown,
-    onRenderLeaf,
-    onRenderParent,
-    ...props
-  }: VirtualizedTreeProps<TLeaf, TParent>) => {
+  innerRef,
```

Comment:
Sooooo I know this goes down a path of personal preferences, but I find without indentation its hard to read and flow the code flow..

Reply:
These files that changed are the exception to the indentation used throughout the codebase. They just weren't caught by the previous-used indent rule. In the interest of not going down the rabbit hole of debating personal preferences, I think we should stick with the convention used in the majority of the codebase.